packages/core-mobile/metro.config.js

```diff
@@ -25,13 +31,22 @@ const baseConfig = {
     }),
     babelTransformerPath: require.resolve('react-native-svg-transformer')
   },
+  watchFolders: [
+    workspaceRoot,
+    nitroCryptoPath,
+  ],
```

Comment:
you shouldn't have to specify watchFolders. we are already doing this via `scripts/generate-metro-monorepo-config.js`.